powerbot.go

```diff
@@ -65,13 +65,19 @@ func (bot *Bot) ParseAndReply(channel string, msg string, user string) {
 			bot.Con.Privmsg(channel, reply)
 			return
 		}
-	} else if code, ok := bot.Commands[command+" "+argument]; ok {
-		write_err := bot.WriteCode(code)
-		var reply string
-		if write_err == nil {
-			reply = fmt.Sprintf("Sent out code %v for %v", code, command)
-		} else {
-			reply = fmt.Sprintf("Error writing to device: %v", write_err)
+	} else if codes, ok := bot.Commands[command+" "+argument]; ok {
+		successful_codes := []int{}
+		failed_codes := map[int]error{}
+		for _, code := range codes {
+			if write_err := bot.WriteCode(code); write_err != nil {
```

Comment:
Your intuition is correct, experimentally I have to wait 1s before sending another code. Please add a sleep.